src/components/common/copyLinkButton/CopyLinkButton.tsx

```diff
@@ -44,12 +51,11 @@ export const CopyLinkButton = ({
           data-qa={dataQa}
           onClick={handleClick}
           type="outline"
-          icon={copiedToClipboard ? <CopiedLinkIcon /> : <CopyLinkIcon />}
+          icon={<CopyLinkIcon />}
```

Comment:
Now that we've removed copiedToClipboard from here, I think we can remove it altogether from the component